com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -37,8 +51,10 @@ private static class Profiling
         internal RenderingMode actualRenderingMode { get { return GL.wireframe || m_DeferredLights == null || !m_DeferredLights.IsRuntimeSupportedThisFrame() || m_DeferredLights.IsOverlay ? RenderingMode.Forward : this.renderingMode; } }
         internal bool accurateGbufferNormals { get { return m_DeferredLights != null ? m_DeferredLights.AccurateGbufferNormals : false; } }
         internal bool usesRenderPass;
+        public DepthPrimingMode depthPrimingMode { get { return m_DepthPrimingMode; } set { m_DepthPrimingMode = value; } }
```

Comment:
Needs public API docs.

Reply:
Added through a <summary> tag. Is this sufficient ?